processing_r/processing/r_templates.py

```diff
@@ -0,0 +1,159 @@
+class RTemplates:
```

Comment:
Can you please:

1. Add a license header to this file
2. Add some docstrings throughout?

Reply:
Done.